### ISSUE TITLE
Labels: "pending external feedback" needs to be more granular.

### ISSUE THREAD

**Issue by alice** [w3ctag/meetings#32]

**Comment by lknik** [comment]
Yes, for example, like who? A group? The original poster? Someone else?

**Comment by hober** [comment]
Here's a strawman: replace "Progress: pending external feedback" and "Progress: stalled" with

* "Progress: pending response to TAG feedback" (aka "waiting for OP to get back to us")
* "Progress: pending feedback solicited from third party" (aka "waiting for someone we roped in on this to get back to us"
